src/components/Whiteboard/ElementBehaviors/Resizable/types.ts

```diff
@@ -34,4 +60,15 @@ export type DimensionsHorizontal = {
   width: number;
 };
 
-export type Dimensions = DimensionsVertical & DimensionsHorizontal;
+export type DimensionsBase = DimensionsVertical & DimensionsHorizontal;
```

Comment:
no need to export `DimensionsBase`
```suggestion
type DimensionsBase = DimensionsVertical & DimensionsHorizontal;
```